packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -140,6 +142,11 @@ export default function BlockToolbar( { hideDragHandle } ) {
 						group="other"
 						className="block-editor-block-toolbar__slot"
 					/>
+					<__unstableBlockNameContext.Provider
```

Comment:
This is used to provide a toggle to open the Navigation sidebar for the currently selected Navigation block.

Reply:
Added a picture for folks too. In trunk clicking this opens a list-view modal, but on this branch it should open the new sidebar.

![list view](https://user-images.githubusercontent.com/1270189/157474374-1667a2e8-09eb-41aa-b7df-7f12e023e59b.png)